source/Core/Drivers/BootLogo.cpp

```diff
@@ -23,11 +24,17 @@ void BootLogo::handleShowingLogo(const uint8_t *ptrLogoArea) {
   }
 
   OLED::clearScreen();
-  OLED::refresh();
 }
 
 void BootLogo::showOldFormat(const uint8_t *ptrLogoArea) {
+#ifdef OLED_128x32
+  // Draw in middle
+  OLED::drawAreaSwapped(16, 8, 96, 16, (uint8_t *)(ptrLogoArea + 4));
```

Comment:
Just out of a parametric mindset:
Could we use something like this here (instead of static values):
https://github.com/Ralim/IronOS-dfu/blob/7ab0a57c49e1913b87b49ae458e641aa48adea87/src/oled.c#L168

Reply:
At the moment the whole bootlogo stuff is setup _only_ for that sizing. Which is sort of why I left it like that. 🤔

Reply:
Just curious? 😊

Reply:
Very valid question.
Still on the fence for what to do about the boot logo stuff.
Since limited frames could be messy, but also nice to do static full screen at the least.
But having half-half also feels messy. 

So maybe I just carve out a little more logo space on these devices to have at least some frames of boot logo?

Reply:
Inclined to leave it as-is for the super short term as there are probably bigger issues to solve 🙃